### DISCUSSION TITLE
retire

### DISCUSSION THREAD

**Discussion by maggu2810** [openhab/openhab-core#3500] [Core Maintainers Discussions]
Hi guys,
I retire from my job as openHAB Core maintainer. Please remove my privileges.
I started to "use" openHAB (v1 that time) over five years ago. After a short time I "used" Eclipse SmartHome. And at the end I tried to invest my time for openHAB 2.
Perhaps it has not been the best option for the openHAB community and if the Bnd migration has been a big pain for the community: Sorry -- but that's another story.
When I remember that I started getting to know OSGi back then. And now worked over five years with that technology... A learned a lot!
I would like to say "Thanks a lot" to you and the other guys (not reading this comment) it has been a pleasure to work with you.
I wish you a good time and I hope that the project will remain successful.
Best regards,
Markus

**Comment by cweitkamp** [comment]
Hi Markus,

That will be a very sad moment and a big loss for the community. But I am sure you have your reasons. Maybe there is a little chance to convince you to stay and continue working on the project.

Otherwise **WE** have to thank **you** a lot for spending so much time and effort into openHAB / ESH. You did a great job and helped with your knowledge and patience wherever it was required and possible. You are kind of mentor / instructor for myself regarding OSGi - and much more - topics and I would not be where I am now without you. I guess there are o much more things to say, but I am lost for words ... 

So, heads up, think positive.

Cheers,

Christoph
